impl/http/src/main/java/io/serverlessworkflow/impl/executors/http/AbstractHttpExecutorBuilder.java

```diff
@@ -46,9 +46,6 @@ protected static RequestSupplier buildRequestSupplier(
         return new WithoutBodyRequestSupplier(Invocation.Builder::head, application, redirect);
       case HttpMethod.OPTIONS:
         return new WithoutBodyRequestSupplier(Invocation.Builder::options, application, redirect);
-      case HttpMethod.PATCH:
-        return new WithBodyRequestSupplier(
-            (request, entity) -> request.method("patch", entity), application, body, redirect);
```

Comment:
We need to discuss about `PATCH`, I will write some options tomorrow 🛌🏼

Reply:
In fact, jersey is only used for testing and explicitly added in examples, it is not really a dependency of the SDK